packages/test-fixtures/name-clash.proto

```diff
@@ -7,15 +7,12 @@ message ReservedFieldNames {
 
     // not allowed as object property, should be escaped
     string to_string = 2;
-
-    // reserved for our ADT discriminator, should be escaped
-    string oneofKind = 3;
```

Comment:
I think it will be useful to leave a `string kind` in here, just so that we have clarity about whether it will be "escaped" or not.